web/package.json

```diff
@@ -9,7 +9,7 @@
     "@mui/material": "^5.11.4",
     "@types/jest": "^29.4.0",
     "@types/lodash": "^4.14.191",
-    "@types/node": "^18.15.0",
+    "@types/node": "^20.5.7",
```

Comment:
we need to update the docker for that maybe?

Reply:
It's already using node 20:
```FROM node:20.5 as frontend```

Reply:
yeah just seen that :+1: